stdlib/weakref.pyi

```diff
@@ -41,7 +41,7 @@ _P = ParamSpec("_P")
 ProxyTypes: tuple[type[Any], ...]
 
 class WeakMethod(ref[_CallableT]):
-    def __new__(cls, meth: _CallableT, callback: Callable[[Self], object] | None = None) -> Self: ...
+    def __new__(cls, meth: _CallableT, callback: Callable[[Self], Any] | None = ..., /) -> Self: ...
```

Comment:
```suggestion
    def __new__(cls, meth: _CallableT, callback: Callable[[Self], Any] | None = ...) -> Self: ...
```

Should fix stubtest complaints.

Reply:
Hmm, that causes some Pyright issues, I'm a bit confused on why they shouldn't be consistent.

Reply:
`WeakMethod` is a pure Python class that takes keyword arguments, while `ref` is implemented in C and doesn't take keyword arguments. We should fix the Pyright issues afterwards, possibly using type ignores.

Reply:
Makes sense.

Reply:
Fixed